corpus/convert.py

```diff
@@ -109,25 +114,33 @@ def run(self):
         c.dump(self.out_corpus.get_path())
 
 
-class CorpusToStmJob(Job):
+class CorpusToStmJob(_CorpusToStmJob):
+    pass
+
+
+class CorpusToStmJobV2(Job):
     """
     Convert a Bliss corpus into a .stm file
     """
 
     def __init__(
         self,
-        bliss_corpus,
-        exclude_non_speech=True,
-        remove_punctuation=True,
-        fix_whitespace=True,
-        name="",
-        tag_mapping=(),
+        bliss_corpus: Path,
+        exclude_non_speech: bool = False,
+        non_speech_tokens: Tuple[str, ...] = (),
+        remove_punctuation: bool = False,
+        punctuation_tokens: Tuple[str, ...] = (),
```

Comment:
do you really prefer a variable length tuple over a list with default None?
We could even give some default punctuation tokens here..

(this comment is not included in "request changes")